6-Typescript/server/routes/todos.ts

```diff
@@ -0,0 +1,13 @@
+import express, { Router } from 'express'
+import { TodoController } from '../handlers/todosController'
+import { DB } from '../interfaces/DB'
+const router: Router = express.Router()
+
+export const todosRouter = (db: DB): Router => {
+    const todos = new TodoController(db)
+    router.get('/all', todos.getAllItems)
+    router.get('/get/:id', todos.getItem)
+    router.post('/set', todos.setItem)
+    router.delete('/delete/:id', todos.removeItem)
```

Comment:
plz follow REST guidelines. (read about it online)
for this specific case it should be like:
```suggestion
    router.get('/', todos.getAllItems)
    router.get('/:id', todos.getItem)
    router.post('/', todos.setItem)
    router.delete('/:id', todos.removeItem)
```